source/dub/dub.d

```diff
@@ -804,7 +804,7 @@ class Dub {
 
 		NativePath placement;
 		final switch (location) {
-			case PlacementLocation.local: placement = m_rootPath; break;
+			case PlacementLocation.local: placement = m_rootPath ~ ".dub/packages/"; break;
```

Comment:
Shouldn't we use buildPath for Windows support?

Reply:
It is working fine on windows, as placement is of type NativePath